clusterMembership/dynamicCluster.go

```diff
@@ -14,13 +11,13 @@ type dynamicCluster struct {
 /*
  * Returns a Snapshot of the Cluster Membership state
  */
-func (c dynamicCluster) Members() []string {
+func (c dynamicCluster) Members() []Node {
```

Comment:
yes will fix in a separate pull request, also want to make Members() not make the copy but instead store a list which is updated on add or remove since these should happen less frequently